test.js

```diff
@@ -71,3 +71,11 @@ test('type inference', t => {
 	t.is(fn({argv: ['5'], inferType: true}, {string: ['foo']}).input[0], 5);
 	t.is(fn({argv: ['5'], inferType: true}, {string: ['_', 'foo']}).input[0], 5);
 });
+
+test('stdinOrInput', async t => {
+	const stdin = await execa.shell('echo \'foo bar\' | ./fixture.js');
+	const input = await execa('./fixture.js', ['foo', 'bar']);
+
+	t.is(stdin.stdout, 'meow\nfoo bar\n');
+	t.is(input.stdout, 'meow\nfoo\nbar');
```

Comment:
Is there a big difference in `echo "foo bar" | cli` and `cat file.txt | cli` where the content of `file.txt` is `foo bar`? I assumed, if I test `echo "foo bar" | cli`, then `cat file.txt | cli` will work as well?